charts/gardener/gardenlet/values.yaml

```diff
@@ -195,8 +195,6 @@ config:
 #   backupLeaderElection:
 #     reelectionPeriod: 5s
 #     etcdConnectionTimeout: 5s
-#   featureGates:
-#     UseEtcdWrapper: true
```

Comment:
What is the motivation for this change? Is the feature now beta or GA?
Does it make then sense to write a release note that operators no longer need to specify this feature gate explicitly?

Reply:
feature gate is now GA. Please see release notes (https://github.com/gardener/etcd-druid/releases/tag/v0.28.0)

Reply:
Could you please add a release note of type `breaking operator` that a Gardener operator would need to remove explicit enablement of this feature gate if it is already present in the gardenlet config. 
I myself had to clean up the GA-ed feature gate from our landscapes. :) 

PS: The release note in etcd-druid side is not of type breaking. :)

Reply:
You are right, it should have been marked as a breaking change. This was a miss from our end. I have added a breaking release note for this.

Reply:
Thanks for adding the release note!